css/resume.css

```diff
@@ -0,0 +1,376 @@
+}
+.timeline-2 h2{
+    display: grid;
+    font-size: 20px;
+    padding-top: 10px; 
+    padding-bottom: 10px;
+    padding-left: 40px;
+}
+.timeline-2 p{
+display: grid;
+font-size: 13px;
+padding-top: 10px; 
+padding-bottom: 10px;
+padding-left: 40px;
+}
+
+.description-2{
+display: grid;
+padding-left: 200px;
+padding-top: 100px;
+padding-bottom: 100px;
+padding-right: 20px;
+justify-content: right;
+}
+.training-1{
+    display: flex;
+    background-color:white;
+    height: 350px;
+    margin-left: 400px;
+    margin-right: 400px;
+    justify-content:left;
+    text-align: left;   
+}
+.timeline-training-1 h1{
+    display: grid;
+    text-align: center;
+    font-size: 20px;
+    padding-top: 40px;
+    padding-bottom: 10px;
+    padding-left: 40px;
+    font-size: 20px;
+    color: var(--primary);
+}
+.timeline-training-1 h2{
+    display: grid;
+    font-size: 20px;
+    padding-top: 10px; 
+    padding-bottom: 10px;
+    padding-left: 40px;
+}
+.timeline-training-1 p{
+    display: grid;
+font-size: 13px;
+padding-top: 10px; 
+padding-bottom: 10px;
+padding-left: 40px;
+}
+.course-content-1{
+    display: grid;
+padding-left: 200px;
+padding-top: 100px;
+padding-bottom: 100px;
+padding-right: 20px;
+justify-content: right;
+}
+.training-2{
+    display: flex;
+    background-color:white;
+    height: 350px;
+    margin-left: 400px;
+    margin-right: 400px;
+    justify-content:left;
+    text-align: left; 
+}
+.timeline-training-2 h1{
+    display: grid;
+    text-align: center;
+    font-size: 20px;
+    padding-top: 40px;
+    padding-bottom: 10px;
+    padding-left: 40px;
+    font-size: 20px;
+    color: var(--primary);
+}
+.timeline-training-2 h2{
+    display: grid;
+    font-size: 20px;
+    padding-top: 10px; 
+    padding-bottom: 10px;
+    padding-left: 40px;
+
+} 
+.timeline-training-2 p{
+    display: grid;
+    font-size: 13px;
+    padding-top: 10px; 
+    padding-bottom: 10px;
+    padding-left: 40px; 
+}
+.course-content-2 p{
+    display: grid;
+    padding-left: 200px;
+    padding-top: 100px;
+    padding-top: 100px;
+    padding-right: 20px;
+    justify-content: right; 
+}
+.grid-skills{
+    display:grid;
+    background-color:white;
+    height: 500px;
+    margin-left: 400px;
+    margin-right: 400px;
+    justify-content:left;
+    text-align: left; 
+}
+.skills{
+    display:grid;
+    }
+.skills-name h1{
+display:flex;    
+padding-top: 70px;
+padding-left: 100px;
+font-family: Verdana, Geneva, Tahoma, sans-serif;
+font-size: 25px;
+}
+.skills p{
+    display:flex;
+    padding-top: 10px;
+    padding-left: 50px;
+
+}
+.skills-name-p-1{
+display: flex;
+    justify-content: space-around;
+    padding-left: 20px;
+}
+.skills-name-p-2{
+    display: flex;
+    justify-content: space-around;
+    padding-left: 50px;
+}
+.languages{
+    display: grid;
+}
+.languages{
+    display:grid;    
+
+}
+.languages h1{
+    padding-left: 100px;
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
```

Comment:
Noticed a couple usages of fonts other than the two mentioned in the spec. My understanding is that Poppins is to be used for headings and Quicksand for the body text